catalog/ui/catalog-ui-search/src/main/webapp/component/workspaces-templates/workspaces-templates.less

```diff
@@ -2,7 +2,7 @@
   .customElement;
 
   .changeBackground(@backgroundAccentContent);
-  height: @minimumLineSize + 2*@minimumButtonSize + 2*@minimumSpacing + 2*@minimumSpacing + 3*@minimumButtonSize;
+  height: 2.5 * @minimumButtonSize + @minimumSpacing;
```

Comment:
✏️ The original simplifies to `@minimumLineSize + 5*@minimumButtonSize + 4*@minimumSpacing`. If this is supposed to be different, you could still use the simplified form for the two lines below. Why `2.5` though?

Reply:
The `2.5 * @minimumButtonSize` came from the height expression used to position the `.home-templates-choices-choice-preview` which is the height of the `Blank`, `Local`, `Federated`, ... template buttons.

The `@minimumSpacing` was used to give a slight buffer so that things display correctly across various theme settings (zoom level, `compact` vs `comfortable`, etc)